domain/domain_test.bzl

```diff
@@ -0,0 +1,14 @@
+load("//:oppia_android_test.bzl", "oppia_android_test")
+
+def domain_test(name, src, test_class):
```

Comment:
Suggest adding a TODO & issue to remove once we can rely on oppia_android_test everywhere.

Reply:
Sounds good!